tutorials/napps/development_environment_setup.rst

```diff
@@ -138,29 +138,36 @@ Cloning existing projects
 If you want to contribute with a Kytos project by coding, you must clone/fork
 its repository located at `GitHub group <https://github.com/kytos>`_. Below, we
 are going to clone all major repositories (python-openflow, kytos-utils
-kytos-napps and kytos):
+and kytos):
 
 .. code-block:: bash
 
-  $ for project in python-openflow kytos-utils kytos kytos-napps; do
+  $ for project in python-openflow kytos-utils kytos; do
```

Comment:
Btw, there is also this one that can be solved together with the previous one:
https://github.com/kytos/documentation/issues/58